pages/release-notes.mdx

```diff
@@ -7,14 +7,17 @@ import { Steps } from "nextra/components"
 
 <Steps>
 <Step>
-## November 2nd 2023 - v0.73.0
+## 2 November 2023 - v0.73.0
 
-#### 🚨 Generate seeds change
+#### 🚨 Generate seeds breaking change 🚨
 
-We've made a change to the way Snaplet generates seeds.
+We've once again made a change to the way Snaplet generates seeds, which breaks previous data generations. Again. We're sorry about this, and while this functionality is still in Beta, we're cognisant of the impact breaking changes have on our users, and our intention is to minimize disruption.  
 
-We previously introduced a hash based on the plan's inputs to our seeding mechanism, but after many feedbacks, it's clear that it's not the right approach. We've decided to remove it and go back to the previous behavior.
-The issue with this approach is that if you alter your plan, the modification will impact all the models involved in the plan.
+We previously introduced a hash based on the plan's inputs to our seeding mechanism, but after user feedback, it's clear that it's not the right approach. The issue with this approach is that if you alter your plan, the modification will impact all the models involved in the plan.
+
+As such, we've decided to roll back it and revert back to the previous behavior.
```

Comment:
Seems like you're this sentence is telling the same thing twice. If we rollback, it means we revert back to the previous behavior. 🤔
Also, isn't "roll it back" more correct than "roll back it"?

Reply:
Fixed! Thanks for the feedback.